src/openeo_test_suite/tests/workflows/conftest.py

```diff
@@ -0,0 +1,50 @@
+import openeo
+import pytest
+import os
+
+@pytest.fixture
+def auto_authenticate() -> bool:
+    """
+    Fixture to act as parameterizable toggle for authenticating the connection fixture.
+    Allows per-test/folder configuration of auto-authentication.
+    """
+    return True
+
+
+@pytest.fixture
+def bounding_box_small(
+    west=10.47, east=10.48, south=46.12, north=46.13, crs="EPSG:4326"
+) -> dict:
+    spatial_extent = {
+        "west": west,
+        "east": east,
+        "south": south,
+        "north": north,
+        "crs": crs,
+    }
+    return spatial_extent
+
+
+@pytest.fixture
+def temporal_interval(interval=["2019-06-01", "2019-07-01"]):
+    return interval
+
+
+@pytest.fixture
+def temporal_interval_one_day(interval=["2019-06-03", "2019-06-05"]):
+    return interval
+
+
+@pytest.fixture
+def s2_stac_url(url="https://planetarycomputer.microsoft.com/api/stac/v1/collections/sentinel-2-l2a"):
+    return url
+
+
+@pytest.fixture
+def t_dim(t_dim="time"):
+    return t_dim
+
+
+@pytest.fixture
+def b_dim(b_dim="band"):
+    return b_dim
```

Comment:
At this point, I don't think it's very useful to define these simple constructs as fixtures, unless I'm missing something.

for example: you define them with default argument values (`def temporal_interval(interval=["2019-06-01", "2019-07-01"]):`, but you can not call these functions explicitly , so they don't allow customization this way

Reply:
The idea was that, since dimension names could be back-end dependent, it would be enough to rename it in one place instead of in every call using it. What would be the best approach to achieve this instead? global variables? A config file?